js/ui/components/discoveryFeed.js

```diff
@@ -38,12 +39,16 @@ var DiscoveryFeed = new Lang.Class({
         Main.discoveryFeed = null;
     },
 
+    notifyHideAnimationCompleted: function() {
+        this.proxy.notifyHideAnimationCompletedRemote();
+    },
+
     callShow: function(timestamp) {
         this.proxy.showRemote(timestamp);
     },
 
-    callHide: function(timestamp) {
-        this.proxy.hideRemote(timestamp);
+    callHide: function() {
+        this.proxy.hideRemote();
```

Comment:
Why did you remove this timestamp?

Reply:
Must have bee a mistake when I removed the timestamp from the old `hidden` method. I'll create a new PR to revert this change - thanks for spotting